motr/idx_dix.c

```diff
@@ -491,7 +489,7 @@ static void cas_get_ast(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 
 	M0_ENTRY();
 	cas_req_prepare(dix_req, &idx, oi);
-	rc = m0_cas_get(creq, &idx, oi->oi_keys);
+	rc = m0_cas_versioned_get(creq, &idx, oi->oi_keys);
```

Comment:
With this patch to enable versioned CAS, m0_cas_get() will only be used to meta operations.
So we can add an assertion in `m0_cas_get()` to assure that it is only called on meta operation.

Reply:
`m0_cas_get` is still used in the UTs at the moment. If that's removed, I can add that assert

Reply:
OK.